app/models/watched_word.rb

```diff
@@ -8,9 +8,10 @@ def self.actions
       censor: 2,
       require_approval: 3,
       flag: 4,
+      link: 8,
```

Comment:
When it is a `link`, should we verify on the server side that URL is valid? Do we only support URLs with HTTP protocol or are other protocols also supported?